### ISSUE TITLE
Signup/Login UI-Server Integration

### ISSUE THREAD

**Issue by amitbinu** [MohamedBengezi/Lift#17]
The logic and methods to call the server for user signup and logic is already done in issue #14 (file:context/AuthContext.js)

This issue should change the existing methods in context/AuthContext.js to call the server with right parameters.

File: https://github.com/MohamedBengezi/Lift/blob/main/src/context/AuthContext.js#L17

This issue is dependent on #16 and #14 

**Branch to use**: Signup/Login
